frontend/app.vue

```diff
@@ -26,3 +38,13 @@ useHead({
     <NuxtPage />
   </NuxtLayout>
 </template>
+
+<style>
+  body {
+    /* Prevent text selection */
```

Comment:
This behavior makes sense for a mobile app--we don't want to be able to select *everything*. But maybe we want things to be selectable by default on a pc?

Reply:
I updated it to have text selectable on any non-touchscreen device, so computers can select and mobile devices won't be able to